tests/delete-user.fg

```diff
@@ -0,0 +1,46 @@
+# The login API is mounted in the odin view as a special
```

Comment:
Let's not call this test file `delete-user.fg`. It expires the user, it doesn't delete the user. The name seems wrong